astropy/table/bst.py

```diff
@@ -443,12 +443,10 @@ def _same_prefix(self, val, node, lst):
         return lst
 
     def __str__(self):
-        if self.root is None:
-            return 'Empty'
-        return self._print(self.root, 0)
+        return repr(self)
 
     def __repr__(self):
-        return str(self)
+        return f'<{self.__class__.__name__}>'
```

Comment:
Maybe just remove `__repr__` as well if we're not using any information? The class name will be given by default...

Reply:
I don't like the default repr that has (in this case) useless object address info. This current version makes the repr consistent with the others.

Reply:
Also, basically the `BST` class should not be used by anybody in practice. I updated the docs accordingly.

Reply:
OK, fine to leave it as you have it.